mkdocs/docs/HPC/jupyter.md

```diff
@@ -0,0 +1,129 @@
+# Jupyter notebook
+
+## What is a Jupyter notebook
+
+A [Jupyter notebook](https://jupyter.org/) is an interactive, web-based environment that allows you to create documents that contain live code, equations, visualizations, and plaintext. The code blocks in these documents can be used to write Python, Java, R and Julia code, among others. The combination of code executions with text and visual outputs make it a useful tool for data analysis, machine learning and educational purposes.
+
+## Using Jupyter Notebooks on the HPC
+
+### Starting a notebook using the web portal
+
+Through the web portal you can easily start a Jupyter notebook on a workernode, via the *Jupyter Notebook* button under the *Interactive Apps* menu item.
+
+<center>
+![image](img/ood_start_jupyter.png)
+</center>
+
+### Importing libraries
+
+Importing libraries in a notebook running on the HPC isn't as straight forward as on notebooks running on your local machine. You can only import libraries that are part of certain modules on the HPC. To find the right module, first look at the toolchain used by the Jupyter notebook version. This can be found when looking at the `JupyterNotebook version` field when creating a new Jupyter notebook session. In the image above `7.2.0` is the notebook version and `GCCcore 13.2.0`is the toolchain used.
+
+After checking the toolchain, you can search the correct module using a shell environment. you can do this by clicking on `Clusters`>`_login Shell Access` in the web portal.
+
+<center>
+![image](img/ood_jupyter_open_shell.png)
```

Comment:
Trim down this image to only show the part on top that really matters, and/or point to https://docs.hpc.ugent.be/web_portal/#shell-access (but that needs an update too it seems)